jupiterone/internal/client/generated.go

```diff
@@ -401,6 +402,27 @@ func (v *CreateInlineQuestionRuleInstanceCreateQuestionRuleInstance) GetOperatio
 	return v.Operations
 }
 
+// GetLabels returns CreateInlineQuestionRuleInstanceCreateQuestionRuleInstance.Labels, and is useful for accessing the field via an interface.
+func (v *CreateInlineQuestionRuleInstanceCreateQuestionRuleInstance) GetLabels() []CreateInlineQuestionRuleInstanceCreateQuestionRuleInstanceLabelsRuleInstanceLabel {
+	return v.Labels
+}
+
+// CreateInlineQuestionRuleInstanceCreateQuestionRuleInstanceLabelsRuleInstanceLabel includes the requested fields of the GraphQL type RuleInstanceLabel.
+type CreateInlineQuestionRuleInstanceCreateQuestionRuleInstanceLabelsRuleInstanceLabel struct {
+	LabelName  string                 `json:"labelName"`
+	LabelValue map[string]interface{} `json:"labelValue"`
```

Comment:
The gql type for `labelValue` is `JSON` because we allow string/boolean/int for the value. However, there is a custom binding in the gql client code generator that turns JSON into this type:
  JSON:
    type: map[string]interface{}

So then labelValues are expected to be objects which we don't want to allow